libcontainer/selinux/selinux.go

```diff
@@ -275,7 +278,9 @@ func mcsAdd(mcs string) error {
 }
 
 func mcsDelete(mcs string) {
+	mcsLock.Lock()
 	mcsList[mcs] = false
+	mcsLock.Unlock()
```

Comment:
Reason I did it was because of one exit path vs multiple. There is a cost to defer which could be avoided in simple cases.

Sent from my iPhone

> On Apr 10, 2016, at 11:37 AM, Marcos Nils <notifications@github.com> wrote:
> 
> In libcontainer/selinux/selinux.go:
> 
> >  	mcsList[mcs] = false
> > +	mcsLock.Unlock()
> In line https://github.com/opencontainers/runc/pull/735/files#diff-0ff20b2f0ffb08e6b40c79c94e4c8e65R283 defer is being used to unlick. Maybe it'd be nice to have a consistent way of unlocking?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>